hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/viewfs/TestViewfsFileStatus.java

```diff
@@ -148,9 +148,11 @@ public void testListStatusACL() throws IOException {
         if (status.getPath().getName().equals("file")) {
           assertEquals(FsPermission.valueOf("-rwxr--r--"),
               status.getPermission());
+          assertEquals(false, status.isDirectory());
```

Comment:
assertFalse() ?

Reply:
Oops, thats a copy paste thing :-). Will fix it.